prac_05/hex_colours.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404/CP5632 Practical
+Hex Colours Dictionary Lookup
+"""
+
+colour_codes = {
```

Comment:
Does this follow the naming convention for dictionaries?
Have a look through the week 5 ppt slides